cypress/e2e/opening-hours-editor.cy.ts

```diff
@@ -0,0 +1,298 @@
+const branchTitle = "Test branch";
+const formattedSearchString = branchTitle.toLowerCase().replace(/ /g, "+");
```

Comment:
Consider inlining this top level variable.

1. It is only used once within a single function.
2. At the top level the name I do not think the variable name makes much sense.